packages/composables/src/use-items.ts

```diff
@@ -305,3 +305,11 @@ export function useItems(collection: Ref<string | null>, query: ComputedQuery):
 		}
 	}
 }
+
+function getEndpoint(collection: string): string {
+	if (collection.startsWith('directus_')) {
```

Comment:
This one's still using the prefix

Reply:
mentioned this one in the review notes, but should also be fixed by creating a single source of truth in a different package 😄